### ISSUE TITLE
[Ontology] changeRequest: "status" (bookable / queued / nonbookable) in BookingOption

### ISSUE THREAD

**Issue by LauraWeissgerber** [IATA-Cargo/ONE-Record#172]
We show in our BookingOptions bookable options as well as nonbookable options.
In the current specification I do not see an option to show, if the BookingOption is bookable or not.

Or can I use bookingStatus for this? But in fact this is not a bookingStatus, cause it is just a bookingOption which was not booked so far.

Proposal 1:
New field "status" in "BookingOptionRequest" with the value "bookable" or "nonbookable".

Porposal 2:
New field  "bookable" (boolean: true/false) in BookingOption to show if a BookingOption is bookable or nonbookable.

Do you see any alternative? What do you think?

**Comment by LauraWeissgerber** [comment]
As an addition:

We would use proposal 1 and rename "bookingStatus" in "status".
With the value:
- bookable
- queued
- nonbookable

In this case the consumer / customer would directly know if the bookingOption is directly bookable, queued (has to be confirmed) or nonbookable.

What do you think about it?

The name "bookingStatus" should just be used in "Booking", NOT "BookingOptionRequest".
"bookingStatus" would have other values (e.g. confirmed).

**Comment by lambertciata** [comment]
On that topic, during MCD API design group we concluded:

```
Carrier’s objective is to provide quality Booking Options. When a carrier provides a Booking Option, is it supposed to be complete so it is not supposed to be updated afterwards. Updates lead to either new Booking Option(s) ot direct update of the Booking (by carrier only)

Milestones - Minimum set of milestones include:
-	BookingOption: Confirmed, queued/pending, unconfirmed (= we won’t look further)
-	Booking: Queued, Confirmed, Rejected
-	Milestones on BookingOptionRequest and BookingRequest don’t seem to be really required.
```

That means the end goal is to propose the best bookable BookingOption based on the BookingOptionRequest data.  Adding "Bookable" or "Nonbookable" in the status would bring confusion in my opinion. If we wish to add this information I would rather add a boolean indicator Bookable:true/false. To be discussed further with taskforce.

**Comment by lambertciata** [comment]
closed with 3.0.0
